doctor.py

```diff
@@ -2,19 +2,96 @@
 
 import argparse
 from collections.abc import Sequence
+from concurrent.futures import ThreadPoolExecutor
+from pkgutil import walk_packages
 
-help = "Diagnose common issues."
+help = "Diagnose and attempt to fix common issues."
```

Comment:
Make a separate `fix` subcommand (which shares much of its code with the `diagnose` subcommand). There are more than a few times where readonly diagnosis is much more appropriate.